gamestonk_terminal/common/newsapi_view.py

```diff
@@ -24,14 +25,19 @@ def news(
         date to start searching articles from formatted YYYY-MM-DD
     show_newest: bool
         flag to show newest articles first
+    sources: str
+        sources to exclusively show news from
     """
-    # TODO: Add argument to specify news source being used
-
-    response = requests.get(
-        f"https://newsapi.org/v2/everything?q={term}&from={s_from}"
-        f"&sortBy=publishedAt&language=en&apiKey={cfg.API_NEWS_TOKEN}",
+    link = (
+        f"https://newsapi.org/v2/everything?q={term}&from={s_from}&sortBy=publishedAt&language=en"
+        f"&apiKey={cfg.API_NEWS_TOKEN}"
     )
 
+    if sources:
+        link += f"&domains={sources}"
```

Comment:
Could we check that each source ends in a .com, and if not add it, this way one could just do `news -s yahoo`

Reply:
Sure